source/_integrations/iammeter.markdown

```diff
@@ -13,12 +13,13 @@ ha_platforms:
   - sensor
 ---
 
-`iammeter` provides real-time readings of single-phase (WEM3080, WEM3162) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com) over Wi-Fi.
+`IAMMETER` provides real-time readings of single-phase (WEM3080) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com/docs/homeassistant) over Wi-Fi.
 
 Example Lovelace Dashboard:
 
 <p class='img'>
-  <img src='/images/integrations/iammeter/demo.jpg' />
+<img src='/images/integrations/iammeter/energy-monitor-Home Assistant.png' />
+<img src='/images/integrations/iammeter/solar-pv-monitoring-home-assistant.png' />
 </p>
```

Comment:
- Please use markdown to include images
- The first filename has spaces, please make it formatted like the second filename
- The first screenshot just shows the Home Assistant energy dashboard, how is that related to this integration?
- The second screenshot is a custom configuration, this is misleading as the end-user might think they will end up getting that when setting up this integration, which is not true.

Reply:
Please use markdown to include images (**got it**)
The first filename has spaces, please make it formatted like the second filename (**got it**)

The first screenshot just shows the Home Assistant energy dashboard, how is that related to this integration?
The second screenshot is a custom configuration, this is misleading as the end-user might think they will end up getting that when setting up this integration, which is not true.

(**In fact, these two pics are related to this  forum post, https://community.home-assistant.io/t/monitor-your-solar-pv-system-in-home-assistant-energy-dashboard-vs-energy-management/342710
I added this in the first commit and removed it according to your suggestion in the second commit.
Without the introduction of the post, these two pics are indeed easy to mislead the user.
Could I add a section at the last of this doc called "How to monitor your energy consumption" and put these two pics and the forum link there?**)

Reply:
If you want to guide users on how to set up energy monitoring with Home Assistant, we should link to that specific page in Home Assistant.

The energy dashboard screenshot is not a feature of this integration, it is a Home Assistant feature. It doesn't belong here.

Reply:
understood,I have already remove such pics in this doc